.builds/freebsd.yml

```diff
@@ -15,7 +15,7 @@ environment:
 tasks:
 - setup: |
     cd Nim
-    git clone --depth 1 -q https://github.com/nim-lang/csources.git
+    git clone --depth 1 -q https://github.com/nim-lang/csources_v1.git csources
```

Comment:
use this instead:
* add a tag v1.0.0 (or whichever is appropriate) in https://github.com/nim-lang/csources_v1
* add a file `$nim/nimdeps.sh` in nim repo containing:
```nim
export NIM_CSOURCES_HASH=v1.0.0
```
* ditto for windows if windows needs different syntax, under `$nim/nimdeps.bat`
* refer to it in each CI/other bash script:
```nim
source nimdeps.sh
git clone --depth 1 --branch $NIM_CSOURCES_HASH https://github.com/nim-lang/csources_v1.git
```

Let's not repeat the same mistakes again, we need $NIM_CSOURCES_HASH checked in nim's repo to allow tooling to automatically know which csources to build from